tests/e2e/json_test.go

```diff
@@ -93,6 +93,14 @@ var _ = Describe("odojsonoutput", func() {
 			Expect(areEqual).To(BeTrue())
 
 		})
+
+		// odo app delete -o json
+		It("should be able to delete app", func() {
+			runCmdShouldPass("odo app create app-deletion-test")
+			// validating that it ran with exit status 0
+			runCmdShouldPass("odo app delete app-deletion-test -o json")
```

Comment:
Outcome of this command is not being validated except ```exit status 0```

Reply:
check https://github.com/redhat-developer/odo/blob/master/tests/e2e/e2e.go#L43

Reply:
@amitkrout can you take a look at the mentioned code and see its clear

Reply:
```
$ ./odo app delete -h
Delete the given application

Usage:
  odo app delete [flags]

[...}

Flags:
  [...]
  -o, --output string    Specify output format, supported format: json
      --project string   Project, defaults to active project
[...]
```
The test step is not verifying the output json. So i am saying capture the result in variable and validate the json format

Reply:
Please read the issue, there is no output only exit status 0, this is the first step 
https://github.com/redhat-developer/odo/issues/1383

Reply:
Something like 
```
actual := runCmdShouldPass("odo app delete app-deletion-test -o json")
desired := `{}`
areEqual, _ := compareJSON(desired, actual)
Expect(areEqual).To(BeTrue())

Reply:
> Please read the issue, there is no output only exit status 0, this is the first step
#1383

Thanks for the clarification.